enterprise/utils.py

```diff
@@ -1390,6 +1433,7 @@ def enroll_users_in_course(
         enrollment_reason=None,
         discount=0.0,
         sales_force_id=None,
+        license_uuids=None,
```

Comment:
Where do you actually use this modified function ? Did you modify it and not call it as future proofing?